.travis.yml

```diff
@@ -3,7 +3,7 @@ language: scala
 env:
   global:
 
-script: sbt test:compile "hot -psource=scalap -w1 -f1" "micro/jmh:run -w1 -f1"
+script: sbt test:compile "hot -psource=scalap -w1 -f1" "; project compilation ; +test" "micro/jmh:run -w1 -f1"
```

Comment:
The `@Fork` annotation issue has been fixed, I can change the CI to run dotty benchmarks in  CI too.